README.md

```diff
@@ -125,15 +128,24 @@ If your Android SDK doesn't live at `$HOME/Android/Sdk` you will need to set
 $ export ANDROID_SDK_ROOT=/path/to/Android/Sdk
 ```
 
-If for some reason your `build-tools` and/or `NDK` also live in different
-locations, you can also configure the following two environment variables:
+:::note Some users have experienced difficulties with cross-compilation
+artefacts missing from the newest NDK, which is downloaded by default in the
+installation process.  If you experience errors of this kind, you may have to
+manually downgrade or install multiple NDK versions as [shown
+here])(img/ndk_downgrade.png) in the Android Studio installer (screengrabbed
+from an Ubuntu installation). Alternately, running all or parts of the
+[install_android_dependencies.sh](install_android_dependencies.sh) script may be
+helpful.
+
+If your `build-tools` and/or `NDK`  live in different locations than the default ones inside /SDK/, or if you want to specify a specific NDK or build-tools version, you can manually configure the following two environment variables:
 
 ```bash
-$ export ANDROID_TOOLS=/path/to/build-tools
-$ export ANDROID_NDK_HOME=/path/to/ndk
+$ export ANDROID_TOOLS=/path/to/SDK/ndk/XX.X.XXXXX/
+$ export ANDROID_NDK_HOME=/path/to/SDK/XX.X.XXXXX/build-tools/XX.X.X/
```

Comment:
I think this is what the env variables are supposed to be according to the DIDKit Makefile.
```suggestion
$ export ANDROID_TOOLS=/path/to/SDK/build-tools/XX.X.X/
$ export ANDROID_NDK_HOME=/path/to/SDK/ndk/XX.X.XXXXX/
```